project/plugins.sbt

```diff
@@ -1,4 +0,0 @@
-addSbtPlugin("org.xerial.sbt" % "sbt-sonatype" % "1.1")
-addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.4")
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
```

Comment:
What's the reason for removing these? Without `sbt-assembly`, we cannot build the project. It may be dependent on some of the above plugins as well.